addons/website/views/snippets/s_adventure.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_adventure" name="Adventure">
+    <section class="s_adventure pt200 pb200 oe_img_bg" style="background-image: url('/web/image/website.s_adventure_default_image'); background-position: 50% 50%;">
+        <div class="o_we_bg_filter bg-white-50"/>
+        <div class="container">
+            <h1 style="text-align: center;" class="display-4">Embark on your</h1>
+            <h2 style="text-align: center;" class="display-1">Next Adventure</h2>
```

Comment:
```suggestion
            <h1 class="display-1" style="text-align: center;">
                <span class="display-4-fs">Embark on your</span>
                <br>Next Adventure
            </h1>
```
The `h1` must include the entire title for Google indexation, otherwise "Embark on your" will be indexed without "Next Adventure".
After this change, the text will move down a bit, so you might need to adjust the top padding of the section to `pt168` to have the same result than before ;).

Reply:
This was done on purpose to make editing those two lines easier, otherwise it's a bit tricky: https://github.com/odoo/odoo/pull/175380#discussion_r1705731940
I did the test of creating the same type of title from scratch in the WE and the result looks like what you suggest, so I guess it's okay :) I've updated it 👍